src/lib/core/hooks/findOutputEntity.ts

```diff
@@ -8,7 +8,7 @@ export function useFindOutputEntity(
   // need to add string at Record's Type due to valueSpecConfig
   dataElementVariables: Record<
     string,
-    VariableDescriptor | string | boolean | undefined
+    VariableDescriptor | string | boolean | unknown | undefined
```

Comment:
If you agree with my suggestion to make `currentPlotFilters` a top-level property of the `VisualizationDescriptor`, you'll be able to do away with the `unknown`.

```suggestion
    VariableDescriptor | string | boolean | undefined
```

Reply:
@jtlong3rd by complying with your suggestions, yes I don't need to define unknown in there so removed it.